src/instructlab/tax/taxonomy.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Third Party
+import click
+
+# First Party
+from instructlab import config
+
+# Local
+from .diff import diff
+
+
+@click.group(chain=True)
+@click.version_option(package_name="cli")
+@click.option(
+    "--config",
```

Comment:
It makes me sad to see that each time with add a group we have to re-add this, isn't there any way not to rewrite that part each time?

Reply:
click weirdness, this flag needs to be at the resource level in order to pass the context down properly